pages/docs/concepts/producer.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Producer
+weight: 3
+---
+
+## What is a Producer? 
+
+An event indicates a state change or update that is triggered by a user's/device's action; this could be something like an item being placed in a shopping cart on an e-commerce website, clicking the subscribe button on a YouTube channel, or a temperature change in a sensor. A producer is a type of application that senses that state change and then publishes that event as a message.
```

Comment:
```suggestion
An event indicates a state change or update that is triggered by a user's/device's action; this could be something like an item being placed in a shopping cart on an e-commerce website, clicking the subscribe button on a YouTube channel, or a temperature change in a sensor. 

A producer is a type of application that senses that state change and then publishes that event as a message.
```

I just suggest to separate it in 2 paragraphs, as section is called `What is a procuder` and final definition is kinda hidden as last sentence in a paragraph, wdyt? or many leave as it is, but bold **A producer** ?

Reply:
`this could be something like an item being placed in a shopping cart on an e-commerce website, clicking the subscribe button on a YouTube channel, or a temperature change in a sensor.`

This is a long sentence. Maybe you can try using bullet points? Something like this:

This could be something like:
* An item being placed in a shopping cart on an e-commerce website.
* Clicking the subscribe button on a YouTube channel.
* A temperature change in a sensor.

Reply:
Sweet, thank you! I'll make the changes right away